src/ValueFormatters/TimeFormatter.php

```diff
@@ -17,40 +15,37 @@
  * @author H. Snater < mediawiki@snater.com >
  */
 class TimeFormatter extends ValueFormatterBase {
+
+	/**
+	 * @deprecated since 0.7, use \ValueParsers\TimeParser::CALENDAR_GREGORIAN instead.
+	 */
 	const CALENDAR_GREGORIAN = 'http://www.wikidata.org/entity/Q1985727';
+
+	/**
+	 * @deprecated since 0.7, use \ValueParsers\TimeParser::CALENDAR_JULIAN instead.
+	 */
```

Comment:
I'm fine with pretty much every solution, but this is unrelated to this patch. The two constants in TimeFormatter are not used any more in TimeFormatter with my patch. They need to be deprecated no matter what.